backend/src/routes/projectRouter.ts

```diff
@@ -299,4 +299,15 @@ projects.delete(
   }
 );
 
+//Returns user projects that use a tech
+projects.get("/v1/projects/:userId/tech/:techId", async (req, res, next) => {
```

Comment:
The path parameter after a resource is generally assumed to be the ID of the resource it follows (so in this case, it would be more generally read as "use the name userId as the project ID" which is slightly confusing.

So it's better instead to write this endpoint from the perspective of the user so that it reads like "the user's projects that have the tech ID of 1781n3281n321893n219n321"
```
GET /v1/users/:userId/projects?hasTechId=1781n3281n321893n219n321
```